Install uyuni and adoc ke (#880)
SUSE/doc-susemanager: modules/installation/pages/server-setup.adoc

[WARNING]

> .Third Party Software

Reviewer: ```suggestion
```

====
{productname} is part of the {sle} 4.0 product family and thus compatible with the software shipped with {sles}.

{productname} is a complex system, and therefore installing third party software is not allowed.
Installing monitoring software provided by a third party vendor is allowed only if you do not exchange basic libraries such as SSL, cryptographic software, and similar tools.

> As part of providing product support, SUSE reserves the right to ask to remove any third party software (and associated configuration changes) and then to reproduce the problem on a clean system.

Reviewer: There's some strange whitespace going on in here.

Author: do you mean the non-breaking space in the middle of "{sle} 4.0"?  It's a valid utf-8 char and equals {nbsp}.

.Procedure: {productname} Setup

> . Log in to the {productname} Server and type [command]``yast2 susemanager_setup`` to begin the setup.

Reviewer: ```suggestion
. On the {productname} Server, at the command line, use the [command]``yast2 susemanager_setup`` command to begin setup.
```
